### PR TITLE
covid badge for events

### PR DESCRIPTION
**Pull request by lexidelorey**

### CODE REVIEW

File: src/main/webapp/postScript.js [hunk 1]
```diff
@@ -45,7 +45,7 @@ function createPost(post, userId) {
 
   const imageElement = document.createElement('img');
   imageElement.id = "image";
-  imageElement.height = "200";
+  imageElement.heighth = "200";
```
Comment (ellisvivian) [inline src/main/webapp/postScript.js]: Heighth or height?

File: src/main/webapp/style.css [hunk 2]
```diff
@@ -7,6 +7,11 @@
   --side-bar: #f1f1f1;
 }
 
+img {
+  max-width: 100%;
+  mad-heigh: 100%;
```
Comment (ellisvivian) [inline src/main/webapp/style.css]: mad-heigh or max-height?

File: src/main/webapp/style.css [hunk 3]
```diff
@@ -9,7 +9,7 @@
 
 img {
   max-width: 100%;
-  mad-heigh: 100%;
+  mad-height: 100%;
```
Comment (ellisvivian) [inline src/main/webapp/style.css]: hehe im sorry but it still says mad instead of max

Comment (lexidelorey) [reply src/main/webapp/style.css]: oh my goodness the typos will never end for me

**Review by ellisvivian — DISMISSED** [review]

**Review by ellisvivian — APPROVED** [review]

**Pull request merged**